lib/console/console.rb

```diff
@@ -13,44 +12,94 @@ def greet_user
     print(messages.greet_user)
   end
 
-  def show_options
-    messages.options.each { |option| print(option) }
+  def get_players(players_factory)
+    print_options
+    prompt
+    players = players_factory.create(get_game_choice)
+    game_instructions(players.current_player_symbol)
+    players
+  end
+
+  def request_move(board = nil)
+    prompt
+    get_move
+  end
+
+  def move_summary(move, board, player, oponent)
```

Comment:
I think "opponent" is spelled with double "p"!

Reply:
whoops 🙈